src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,7 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        System.out.println("Hello world!");
```

Comment:
According to the checklist, there should not be an empty line after the class declaration and before the method signature. Please remove the redundant empty line at line 3.